de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v2_0/editor/FSKEditorJSNodeModel.java

```diff
@@ -524,10 +531,55 @@ protected PortObject[] performExecute(PortObject[] inObjects, ExecutionContext e
 //      }
 
       // Collect R packages
-      final Set<String> librariesSet = new HashSet<>();
-      librariesSet.addAll(new RScript(modelScript).getLibraries());
-      librariesSet.addAll(new RScript(visualizationScript).getLibraries());
-      packages = new ArrayList<>(librariesSet);
+      if(metadata != null && SwaggerUtil.getLanguageWrittenIn(metadata) != null) {
+        String languageWrittenIn = SwaggerUtil.getLanguageWrittenIn(metadata);
+        String pythonPath = null; // Path to the Python executable in the conda environment
+        String envPath = null; // Path to the conda environment
+        if(PreferencePage.getPython3CondaEnvironmentDirectoryPath() != null)
+          envPath = PreferenceInitializer.getPython3Env();
+        else
+          envPath = PreferenceInitializer.getPython2Env();
+        
+        if(((FskPortObject)inObjects[0]).overridePackages) {
+          pythonPath =envPath+"/bin/python";
```

Comment:
This path doesn't work for Windows. There it would be more like:
`          pythonPath =Path.of(envPath, "python.exe").toString();
`